vendor/github.com/mongodb/mongo-tools-common/db/namespaces.go

```diff
@@ -97,17 +60,11 @@ func GetIndexes(coll *mongo.Collection) (*mongo.Cursor, error) {
 	return coll.Indexes().List(context.Background())
 }
 
-func getIndexesPre28(coll *mgo.Collection) (*mgo.Iter, error) {
```

Comment:
similar Q to below: this was deleted because GODRIVER-279 has been implemented, right? should we also delete the comment above on GetIndexes?

Reply:
I removed them simply because they're not called anywhere. I'm not really sure what specifically they were used for in the past. I do think I can remove the comment from `GetIndexes` though, so I've done that.